hms-app/src/components/buttons/ParticipateButton.tsx

```diff
@@ -181,18 +185,27 @@ export default function ParticipateButton(props: IProps) {
   }, [ideaData])
 
   return (
-    <Button
-      disabled={buttonIsDisabled}
-      onClick={
-        participantCheck ? removeThisIdeaParticipant : addThisIdeaParticipant
-      }
-      style={{
-        backgroundColor: participantCheck
-          ? LEAVE_BUTTON_COLOR
-          : JOIN_BUTTON_COLOR,
-      }}
-    >
-      {participantCheck ? 'Leave Idea' : 'Join Idea'}
-    </Button>
+    <>
+      <Group>
+        <Switch
```

Comment:
could you add 
color={RELOAD_BUTTON_COLOR}
to the switch to make it green? ( I think green is more intuitiv then blue) 
and we should update the color file to 
export const RELOAD_BUTTON_COLOR = 'teal'